app/routes/gamer-activity.js

```diff
@@ -125,24 +127,28 @@ router.post("/activity", async (req, res) => {
  * @apiGroup Activity
  *
  * @apiParam {String} student_number Student number, 8 digit integer.
+ * @apiParam {String} exec_name Exec that ended the activity.
  *
  * @apiSuccess {Object} gamer_activity Gamer profile object.
  * @apiSuccess {String} gamer_activity.student_number Student number, 8 digit integer.
  * @apiSuccess {Number} gamer_activity.pc_number PC number.
  * @apiSuccess {String} gamer_activity.game Game name.
  * @apiSuccess {String} gamer_activity.started_at Datetime when the activity started.
  * @apiSuccess {String} gamer_activity.ended_at Datetime when the activity ended.
+ * @apiSuccess {string} gamer_activity.exec_name Exec that ended the activity.
  *
  * @apiError {String} 500 Internal server error.
  */
-router.patch("/activity/update/:student_number", async (req, res) => {
-  const ended_at = moment()
-    .tz("America/Los_Angeles")
-    .format("YYYY-MM-DD HH:mm");
-  const { student_number } = req.params;
-
-  const query = `UPDATE ${schema}.gamer_activity
-                   SET ended_at = $1 
+router.patch(
+  "/activity/update/:student_number/:exec_name",
```

Comment:
Do you need add the exec_name in the actual url? I think you can just add it to the body of the request?

Reply:
I added it as a param because we already put student_number in the param and this just makes the change more trivial - we could go with the (arguably better) approach with putting it into the request body because it makes a bit more sense (the body should contain information we're updating the resource with whilst the uri contains information about where to find resources)

Reply:
OK I changed it so we pass it in the body instead, this should be _more_ conforming to what PATCH is supposed to be